### PR TITLE
(doc) document a simple inventory file in the README

### PR DESCRIPTION
**Pull request by adreyer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,10 @@ ssh:
 
 `modulepath`: The module path to load tasks and plan code from. This is a list of directories separated by the OS-specific path separator (`:` on Linux/macOS, `;` on Windows).
 
+`transport`: The transport to use when none is specified in the url(default: `ssh`)
+
+`inventoryfile`: The path to the `inventory.yaml` file(default: `~/bolt/inventory.yaml`
```
Comment (nicklewis) [inline README.md]: This is `~/.puppetlabs/bolt/inventory.yaml`

File: README.md [hunk 2]
```diff
@@ -103,6 +107,37 @@ ssh:
 
 `task-environment`: The environment from which Orchestrator will serve task implementations. (default: `production`)
 
+### Node specific configruation with the inventory file.
+
+The inventory file allows you to group nodes and set up node specific configuration defaults. To set up default for ssh and winrm connections create and inventoryfile at `~/bolt/inventory.yaml` with the folling content.
```
Comment (nicklewis) [inline README.md]: `~/.puppetlabs/bolt/inventory.yaml`

File: README.md [hunk 3]
```diff
@@ -103,6 +107,37 @@ ssh:
...
+  - name: ssh_nodes
+    nodes:
+      - sshnode1.example.com
+      - sshnode2.example.com
+    config:
+      transport: ssh
+  - name: win_nodes
+    nodes:
+      - winnode1.example.com
+      - winndoe2.example.com
```
Comment (nicklewis) [inline README.md]: `winnode2` typo

File: README.md [hunk 4]
```diff
@@ -103,6 +107,37 @@ ssh:
...
+      transport: winrm
+      transports:
+        winrm:
+          ssl: false
+```
+
+You can then leave the transport option off of the url when targeting those nodes.
+
+```
+$ bolt task run package name=puppet action=status --nodes sshnode1.example.com,winndode1.example.com
```
Comment (nicklewis) [inline README.md]: `winnode1` typo (differently!)

File: README.md [hunk 5]
```diff
@@ -103,6 +107,37 @@ ssh:
...
+```
+
+The global tranpsort option and any transport specific config option can be set in the inventory file. You can also set `user` and `password` for specific transports.
```
Comment (nicklewis) [inline README.md]: `transport` typo

File: README.md [hunk 6]
```diff
@@ -103,6 +107,37 @@ ssh:
 
 `task-environment`: The environment from which Orchestrator will serve task implementations. (default: `production`)
 
+### Node specific configruation with the inventory file.
+
+The inventory file allows you to group nodes and set up node specific configuration defaults. To set up default for ssh and winrm connections create and inventoryfile at `~/.puppetlabs/bolt/inventory.yaml` with the following content.
+
+```yaml
+groups:
```
Comment (MikaelSmith) [inline README.md]: I'm not sure this actually works. We validate that every group has a name, and the top-level group here doesn't.

Comment (MikaelSmith) [reply README.md]: Oh, I missed that we provide a default name of `all` for the top-level group. That'll be important on my PR.

Comment (adreyer) [reply README.md]: The top level group is given the name "all" https://github.com/puppetlabs/bolt/blob/master/lib/bolt/inventory.rb#L46

**Comment by MikaelSmith** [comment]
@nicklewis this has been updated, can you re-review?

**Pull request merged**